packages/superset-ui-core/src/utils/isRequired.ts

```diff
@@ -1,3 +1,3 @@
-export default function isRequired(field: string) {
+export default function isRequired(field: string): never {
```

Comment:
hadn't seen `never` before 😮

Reply:
https://stackoverflow.com/questions/37910669/what-is-the-difference-between-never-and-void-in-typescript